src/node/constants.ts

```diff
@@ -0,0 +1,32 @@
+const os = require('os');
+const path = require('path');
+
+// @ts-ignore
+import CSInterface from '../lib/CSInterface';
+
+export const homeDirectory = os.homedir();
+export const csInterface = new CSInterface();
+
+export const macDest = path.join(
+  homeDirectory,
+  'Library',
+  'Application Support',
+  'Adobe',
+  'Plainly Videos',
+);
+
+export const windowsDest = path.join(
+  homeDirectory,
+  'AppData',
+  'Roaming',
+  'Adobe',
+  'Plainly Videos',
+);
+
+const device: string = os.platform();
+
+export function getSettingsDirectory(): string {
+  if (device === 'win32') return windowsDest;
+
+  return macDest;
```

Comment:
`This else clause can be omitted because previous branches break early.`
Which makes sense, becuase it must go under it, no need for `else`. Do you want me to disable this error message?